src/github.com/getlantern/flashlight/config/flags.go

```diff
@@ -89,3 +100,14 @@ func (updated *Config) applyFlags() error {
 
 	return nil
 }
+
+func parseFrontFQDNs(frontFQDNs string) (map[string]string, error) {
+	fqdns := map[string]string{}
+	if err := yaml.Unmarshal([]byte(frontFQDNs), fqdns); err != nil {
+		return nil, err
+	}
+	if len(fqdns) == 0 {
```

Comment:
In the case of the command line, while a logical interpretation, in practice that looks like it would only hide user errors.  When loading from a flashlight.yaml it makes sense, since I imagine in that case the YAML reader will leave that field as the zero value (an empty dict), so that would be common.  So I think I'll do what you say: treat {} the same as nil, _but_ still keep the check here, so you don't accidentally enter an empty dict in the command line.  What do you think?